pom.xml

```diff
@@ -60,7 +60,7 @@
 		<dependency>
 			<groupId>org.webjars</groupId>
 			<artifactId>bootstrap</artifactId>
-			<version>3.3.7</version>
+			<version>4.0.0</version>
```

Comment:
The latest version is 4.1.2.

Reply:
Eclipse threw a bunch of mysterious errors when I tried to upgrade to 4.1.2. But there's a 4.1.3 out there now, and that worked fine. Also got the locator jar working.